tiledb/sm/cpp_api/dimension.h

```diff
@@ -267,6 +271,9 @@ class Dimension {
    */
   std::string tile_extent_to_str() const {
     auto tile_extent = _tile_extent();
+    if (tile_extent == NULL) {
+      return "";
```

Comment:
Should this be "N/A"? I think that was what the ticket was asking for, instead of an empty string.

Reply:
I was thinking about  filling  the "N/A"(or other special chracter) to the tile_extent byte vector. If the dimension type is primitive data type(int32_t,int64_t, etc), what should we do for that? we have tile_extent() method in our cpp_api, it will return default value of the same type T(return T()).

Reply:
Just asked the requester. We will output "N/A" for tile_extent _to_str() for hilbert ordering arrays. The tile_extent() method will output default value of type T for hilbert ordering arrays